### PR TITLE
merge duplicate rules - 1

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [x] Filters maintenance.

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: SpanishFilter/sections/specific.txt [hunk 1]
```diff
@@ -1543,6 +1537,7 @@ universoformulas.com##.post-inner > div[style="margin: 0 0 25px 0"] > span[style
 torrentbrazil.org##a[href^="https://goo.gl/"] > img
 jn.pt##div[class^="t-pub-box-"]
 televalltv.net###floatLayer6
+cinemaniahdd.com##a[href^="//mellowads.com"]
 tufutbolpro.com##a[href^="http://bit.ly/"]
```
Comment (Sergey-Lyapin) [inline SpanishFilter/sections/specific.txt:1541]: @SKEIDs 
It seems you forgot to delete this rule.

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: SpanishFilter/sections/specific.txt [hunk 2]
```diff
@@ -1543,6 +1537,7 @@ universoformulas.com##.post-inner > div[style="margin: 0 0 25px 0"] > span[style
 torrentbrazil.org##a[href^="https://goo.gl/"] > img
 jn.pt##div[class^="t-pub-box-"]
 televalltv.net###floatLayer6
+cinemaniahdd.com##a[href^="//mellowads.com"]
```
Comment (Sergey-Lyapin) [inline SpanishFilter/sections/specific.txt:1540]: @SKEIDs 
It looks like you added this rule by mistake.

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

**Review by SKEIDs — COMMENTED** [review]
I mistook to solve conflicts

File: SpanishFilter/sections/specific.txt (hunk 1 above)
Comment (SKEIDs) [reply SpanishFilter/sections/specific.txt:1541]: I mistook to solve conflicts

File: SpanishFilter/sections/specific.txt (hunk 2 above)
Comment (SKEIDs) [reply SpanishFilter/sections/specific.txt:1540]: I mistook to solve conflicts

**Comment by SKEIDs** [comment]
I will re-make pull request.

**Pull request closed without merging**